sympy/physics/quantum/qubit.py

```diff
@@ -282,10 +282,22 @@ class IntQubitState(QubitState):
     """A base class for qubits that work with binary representations."""
 
     @classmethod
-    def _eval_args(cls, args):
+    def _eval_args(cls, args, **extra_args):
```

Comment:
This should go in the constructor, 

```suggestion
    def _eval_args(cls, args, nqubits=None):
```

Not only is this better for things like `help()` that show the function signature, but it disallows other keyword arguments (like say a misspelled "nquibits"), which with `**extra_args` would just be silently ignored.

Reply:
That's true! Thank you for the feedback, I will fix it.